src/main/java/network/brightspots/rcv/RawContestConfig.java

```diff
@@ -230,6 +243,14 @@ public void setName(String name) {
       this.name = name;
     }
 
+    public List<String> getAliases() {
+      return List.copyOf(aliases);
```

Comment:
Just curious: why a re we returning a `copyOf` here instead of the object itself?

Reply:
This is public, so I'm afraid of a caller assuming they own the List and mutating it. It's not currently used, just added for completeness, so I'd rather be safe than let a usage bug sneak in. I can also just delete the function entirely.